src/main/java/accounting/data/TransactionDAO.java

```diff
@@ -63,7 +65,7 @@ private Transaction getHelper(long id) {
         return transaction;
     }
 
-    private long insertEntryAndUpdateAccounts(long id, Account toAccount, Account fromAccount, double amount) {
+    public long insertEntryAndUpdateAccounts(long id, Account toAccount, Account fromAccount, double amount) {
```

Comment:
This should probably stay private so that people don't accidently use it.  I don't see it being used anywhere else in the code.  But if it's needed somewhere else then it's whatever.